playground/infrastructure/cloudbuild/playground_cd_stable.yaml

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+ # contributor license agreements.  See the NOTICE file distributed with
+ # this work for additional information regarding copyright ownership.
+ # The ASF licenses this file to You under the Apache License, Version 2.0
+ # (the "License"); you may not use this file except in compliance with
+ # the License.  You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+steps:
+  - name: ubuntu
+    args:
+      - '-c'
+      - |-
+        echo "Search for CDLOG keyword to find valuable logs entries"
+        echo "CDLOG $(date --utc '+%D %T') Trigger run inputs: 
+         PR URL: $_PR_URL; 
+         PR Source Repo: $_FORK_REPO; 
+         PR Branch: $_PR_BRANCH;
+         PR Commit: $_PR_COMMIT; 
+         WebHook Action: $_PR_TYPE; 
+         Merged to: $_TARGET_PR_REPO_BRANCH; 
+         Playground DNS: $_DNS_NAME; "
+        if [[ ${_TARGET_PR_REPO_BRANCH} != "apache:master" ]]; then
+            echo "CDLOG Merging not into the master, but into the branch apache/${_TARGET_PR_REPO_BRANCH}. Exiting"
+            exit 0
+        fi
+        
+        if [[ ${_PR_TYPE} == "closed" ]] && [[ ${_MERGE_STATUS} == "true" ]]; then
+
+            echo "CDLOG Pull Request $_PR_URL has been successfully closed and merged into Apache/Beam GitHub repository. Continuing the process."
+              
+            echo "CDLOG Continous Deployment of Playground Examples (CD) in the progress."
+  
+            apt update > /dev/null 2>&1
+  
+            apt install -y git curl > /dev/null 2>&1
+  
+            apt-get install -y apt-transport-https ca-certificates gnupg > /dev/null 2>&1
+  
+            echo "deb https://packages.cloud.google.com/apt cloud-sdk main" > /dev/null 2>&1 | tee -a
+            /etc/apt/sources.list.d/google-cloud-sdk.list > /dev/null 2>&1
+  
+            curl https://packages.cloud.google.com/apt/doc/apt-key.gpg > /dev/null 2>&1 | apt-key add - > /dev/null 2>&1 
+  
+            apt-get update && apt-get install -y google-cloud-sdk > /dev/null 2>&1
+  
+            git clone --branch master https://github.com/apache/beam.git > /dev/null 2>&1
+  
+            cd beam
+  
+            git remote add forked https://github.com/${_FORK_REPO}.git > /dev/null 2>&1
+  
+            curl -O --output-dir /tmp https://raw.githubusercontent.com/apache/beam/master/playground/infrastructure/cloudbuild/examples_cd.sh
```

Comment:
we should fetch this file from the commit from which we are running the build. Otherwise this can lead to very hard to debug problems

Reply:
Also, what's the reason for fetching this file via http if we already have the repo cloned above?

Reply:
No need to fetch for CD, since it runs from master branch